### ISSUE TITLE
Quarkus homepage 404

### ISSUE THREAD

**Issue by Dieken** [quarkusio/quarkus#34552]
### Describe the bug

https://quarkus.io

Seems Quarkus 3.2 released half an hour ago breaks the homepage.

```
Error 404

You have found nowhere.
The pathway to normal space
is found in the nav.

A haiku for you.
```

**Comment by Dieken** [comment]
```
$ curl https://quarkus.io -v
*   Trying 185.199.111.153:443...
* Connected to quarkus.io (185.199.111.153) port 443 (#0)
* ALPN: offers h2
* ALPN: offers http/1.1
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* [CONN-0-0][CF-SSL] (304) (OUT), TLS handshake, Client hello (1):
* [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Server hello (2):
* [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Unknown (8):
* [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Certificate (11):
* [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, CERT verify (15):
* [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Finished (20):
* [CONN-0-0][CF-SSL] (304) (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / AEAD-AES256-GCM-SHA384
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=quarkus.io
*  start date: Jun 19 08:13:11 2023 GMT
*  expire date: Sep 17 08:13:10 2023 GMT
*  subjectAltName: host "quarkus.io" matched cert's "quarkus.io"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* Using HTTP2, server supports multiplexing
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* h2h3 [:method: GET]
* h2h3 [:path: /]
* h2h3 [:scheme: https]
* h2h3 [:authority: quarkus.io]
* h2h3 [user-agent: curl/7.87.0]
* h2h3 [accept: */*]
* Using Stream ID: 1 (easy handle 0x7f910700b800)
> GET / HTTP/2
> Host: quarkus.io
> user-agent: curl/7.87.0
> accept: */*
> 
< HTTP/2 404 
< server: GitHub.com
< content-type: text/html; charset=utf-8
< access-control-allow-origin: *
< etag: "64a573e4-e2"
< x-proxy-cache: MISS
< x-github-request-id: BC48:118C:F7F6:102E1:64A57956
< accept-ranges: bytes
< date: Wed, 05 Jul 2023 14:08:22 GMT
< via: 1.1 varnish
< age: 0
< x-served-by: cache-tyo11927-TYO
< x-cache: MISS
< x-cache-hits: 0
< x-timer: S1688566103.656755,VS0,VE157
< vary: Accept-Encoding
< x-fastly-request-id: 8c7af0e544c6018611e1956ecd3008101281af3f
< content-length: 226
< 
<h1 id="error-404">Error 404</h1>

<h2 id="you-have-found-nowherethe-pathway-to-normal-spaceis-found-in-the-nav">You have found nowhere.<br />The pathway to normal space<br />is found in the nav.</h2>

<p>A haiku for you.</p>
* Connection #0 to host quarkus.io left intact
```

**Comment by tomperr** [comment]
Moving this issue to the [quarkus website repo](https://github.com/quarkusio/quarkusio.github.io) might be relevant

**Comment by bulldog98** [comment]
It's already fixed see #34551
